content/docs/command-reference/remote/modify.md

```diff
@@ -609,6 +624,15 @@ more details.
 
 > Please note our [Privacy Policy (Google APIs)](/doc/user-guide/privacy).
 
+- `gdrive_acknowledge_abuse` - acknowledge the risk of downloading potentially
+  [abusive](https://support.google.com/docs/answer/148505) data. Files
```

Comment:
> potentially abusive ... ? (files, content, something is needed here)

@shcheklein It's "data" here. Not sure why #3810 doesn't have that word... ⏳

Reply:
why not files?

Reply:
> why not files?

Because the next sentence starts with "Files".

Reply:
But OK, we can do this:
```suggestion
- `gdrive_acknowledge_abuse` - acknowledge the risk of downloading potentially
  [abusive](https://support.google.com/docs/answer/148505) files. Anything
```